src/pybind/hmm/tree-accu_pybind.cc

```diff
@@ -0,0 +1,48 @@
+// pybind/tree-accu_pybind.cc
+
+// Copyright (c) 2019, Johns Hopkins University (Yenda Trmal<jtrmal@gmail.com>)
+
+// See ../../COPYING for clarification regarding multiple authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+// See the Apache 2 License for the specific language governing permissions and
+// limitations under the License.
+
+#include "hmm/tree-accu_pybind.h"
+#include "hmm/tree-accu.h"
+
+using namespace kaldi;
+
+void pybind_hmm_tree_accu(py::module& m) {
```

Comment:
Thanks.  Do you want me to merge this soon or wait?
BTW, it's not clear to me that we'd need this part of the code anytime soon.. I would start with tree, hmm, transition-model, posteriors, and functions that operate on them.  Incidentally, in the kaldi10 branch I made some nice simplifications to the tree, HMM, transition-model code; but it never seems to be a good time to merge.  (Breaks back compatibility).